docs/reminders.md

```diff
@@ -0,0 +1,31 @@
+# Reminders
```

Comment:
We will want to make sure that Apollo checks that it has the right thread permissions.

We should also probably run a check that users accessing the channel have permission to see/send messages in threads.

Reply:
We do check thread permissions! If Apollo is not able to create or send a message in a thread, the event organizer is notified via DM of the missing permissions to ensure a reminder is sent in the future.

I don't think there's a permission that regular attendees need (or that we should check). As long as they can view the channel, they should be good. Even if they don't have permission to create threads or send messages in them, they should still be able to view the public thread, which is all that really matters for reminders.

Reply:
Right now, it looks like Discord doesn't have a "view threads" permission, which I have to imagine is going to be part of "see channels / read messages" right now, but it's definitely something we should keep an eye out for. Even if that's just a comment in the codebase saying "if discord ever decides to lock thread viewer permissions, this is where we should check that a user can view threads." 

If we're not going to enforce that users will/should be able to send messages to a thread that they're effectively forced-added to, we should note that in the documentation. Something small like "be sure to check your server's permissions if you'd like your members to be able to send messages in threads that Apollo creates! Or not!"